packages/core/src/session_pool/session_pool.ts

```diff
@@ -58,6 +58,19 @@ export interface SessionPoolOptions {
 
     /** @internal */
     log?: Log;
+
+    /**
+     * If set to true, `SessionPool` will periodically persist its state to the key-value store.
+     * @default true
+     */
+    enablePersistence?: boolean;
+}
+
+/**
+ * Override persistence-related options provided in {@apilink SessionPoolOptions} for a single method call
+ */
+interface PersistenceOptionsOverrides {
```

Comment:
I wanted to have a single interface for both since it's exactly the same. Let's export it and reuse it instead of creating two local implementations.

I also don't like the name very much, will try to come up with something else, I don't have any ideas right now :]

Reply:
I've restructured it a bit, so that the same interface is used in the constructor and `options` parameter. Does that look okay?

I'm also not sure where to actually put this interface in the project hierarchy. I don't see a matching module for utilities/types shared by both statistics and session pool